Directory.Build.props

```diff
@@ -100,7 +100,7 @@
   <PropertyGroup>
     <LangVersion Condition="'$(MSBuildProjectExtension)'=='.csproj'">8.0</LangVersion>
     <LangVersion Condition="'$(MSBuildProjectExtension)'=='.fsproj'">4.7</LangVersion>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
If we leave this as "true" when enabling FxCop Analyzer, then we aren't able to build ML.NET unless we address all of the warnings raised by FxCop. So setting it as false let us build ML.NET in order to run tests locally and see that our changes don't brake anything else.

We'd never want to set this as "false" on the main repository, and it's only meant to be "false" while fixing the errors raised by FxCop.